Library/Formula/when.rb

```diff
@@ -1,12 +1,15 @@
-require 'formula'
-
 class When < Formula
-  homepage 'http://www.lightandmatter.com/when/when.html'
-  url 'http://ftp.de.debian.org/debian/pool/main/w/when/when_1.1.33.orig.tar.gz'
-  sha1 'e314a64d74f79be07da1ade6d39a1fc51205f81d'
+  homepage "http://www.lightandmatter.com/when/when.html"
+  url "http://ftp.de.debian.org/debian/pool/main/w/when/when_1.1.34.orig.tar.gz"
+  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/w/when/when_1.1.34.orig.tar.gz"
+  sha256 "ce0540bde96b361d6d0770803901364a687d971ffedd33e36f8f8bef32b19600"
+  head "https://github.com/bcrowell/when.git"
 
   def install
-    bin.install 'when'
-    man1.install 'when.1'
+    system "make", "prefix=#{prefix}", "install"
+  end
+
+  test do
+    system bin/"when", "--help"
```

Comment:
do you mean `when d`?   All of the regular commands require it to be initialized first, otherwise it blows up:

```
🐙  brew test when
Testing when
==> /usr/local/Cellar/when/1.1.34/bin/when d

/usr/local/Cellar/when/1.1.34/bin/when
d

To set up your calendar, do the command ``when'' in an interactive terminal window.
Error: when: failed
Failed executing: /usr/local/Cellar/when/1.1.34/bin/when d
/usr/local/Library/Homebrew/formula.rb:728:in `block in system'
/usr/local/Library/Homebrew/formula.rb:685:in `open'
/usr/local/Library/Homebrew/formula.rb:685:in `system'
/usr/local/Library/Formula/when.rb:13:in `block in <class:When>'
/usr/local/Library/Homebrew/formula.rb:637:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:17:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:634:in `run_test'
/usr/local/Library/Homebrew/cmd/test.rb:38:in `block (2 levels) in test'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/cmd/test.rb:37:in `block in test'
/usr/local/Library/Homebrew/cmd/test.rb:15:in `each'
/usr/local/Library/Homebrew/cmd/test.rb:15:in `test'
/usr/local/Library/brew.rb:135:in `<main>'
````